mesmerize_napari/tests/test_core.py

```diff
@@ -262,21 +307,70 @@ def test_cnmfe():
     process.wait()
 
     df = load_batch(batch_path)
-    print(df)
     # Confirm output path is as expected
     assert df.iloc[-1]['outputs']['success'] is True
     assert df.iloc[-1]['outputs']['traceback'] is None
-    assert Path(os.path.join(vid_dir,
-        f'{df.iloc[-1]["uuid"]}_cnmf-memmap__d1_60_d2_80_d3_1_order_C_frames_2000_.mmap')) == \
-           get_full_data_path(df.iloc[-1]['outputs']['cnmf-memmap-path']) == \
-          Path(os.path.join(vid_dir, df.iloc[-1]['outputs']['cnmf-memmap-path']
-                       ))
+    assert Path(vid_dir,
+                'memmap__d1_128_d2_128_d3_1_order_C_frames_1000_.mmap') == \
+        get_full_data_path(df.iloc[-1]['outputs']['cnmfe_memmap'])
+    assert Path(tmp_dir, f'{df.iloc[-1]["uuid"]}.hdf5') == \
+        get_full_data_path(df.iloc[-1]['outputs']['cnmfe_outputs'])
 
-    assert Path(os.path.join(vid_dir,
-        f'{df.iloc[-1]["uuid"]}.hdf5')) == \
-        get_full_data_path(df.iloc[-1]['outputs']['cnmf-hdf5-path']) == \
-        Path(os.path.join(vid_dir, df.iloc[-1]['outputs']['cnmf-hdf5-path']))
+def test_remove_item():
+    set_parent_data_path(vid_dir)
+    algo = 'mcorr'
+    df, batch_path = _create_tmp_batch()
+    print(f"Testing mcorr")
+    input_movie_path = get_datafile(algo)
+    print(input_movie_path)
+    df.caiman.add_item(
+        algo=algo,
+        name=f'test-{algo}',
+        input_movie_path=input_movie_path,
+        params=test_params[algo]
+    )
+
+    assert df.iloc[-1]['algo'] == algo
+    assert df.iloc[-1]['name'] == f'test-{algo}'
+    assert df.iloc[-1]['params'] == test_params[algo]
+    assert df.iloc[-1]['outputs'] is None
+    try:
+        UUID(df.iloc[-1]['uuid'])
+    except:
+        pytest.fail("Something wrong with setting UUID for batch items")
 
+    assert get_full_data_path(df.iloc[-1]['input_movie_path']) == \
+                  Path(vid_dir, f"{algo}.tif") == \
+                  Path(vid_dir, df.iloc[-1]['input_movie_path'])
 
-def test_remove_item():
-    pass
\ No newline at end of file
+    df.caiman.add_item(
+        algo=algo,
+        name=f'test1-{algo}',
+        input_movie_path=input_movie_path,
+        params=test_params[algo]
+    )
+
+    assert df.iloc[-1]['algo'] == algo
+    assert df.iloc[-1]['name'] == f'test1-{algo}'
+    assert df.iloc[-1]['params'] == test_params[algo]
+    assert df.iloc[-1]['outputs'] is None
+    try:
+        UUID(df.iloc[-1]['uuid'])
+    except:
+        pytest.fail("Something wrong with setting UUID for batch items")
+
+    assert get_full_data_path(df.iloc[-1]['input_movie_path']) == \
+                  Path(vid_dir, f"{algo}.tif") == \
+                  Path(vid_dir, df.iloc[-1]['input_movie_path'])
+    # Check removing specific rows works
```

Comment:
useful to check that the paths for the batch item data also no longer exist

Reply:
Sorry can you explain this?

Reply:
When a batch item is deleted, check that any items related to that batch item (like mcorr output, cnmf output, corr imgs etc.) are deleted.

Reply:
got it!